tools/debugging/stress_test_transfers.py

```diff
@@ -196,8 +201,6 @@ def wait_for_status_ready(base_url: str, retry_timeout: int) -> None:
     while not is_ready(base_url):
         gevent.sleep(retry_timeout)
 
-    raise RuntimeError("Stopping")
```

Comment:
This was a bug, in the previous re-factoring I missed that this ended up always being `raised`. I believe this was actually dead code before the refactoring.